### PR TITLE
MM-48421 new DAU/MAU models

### PR DESCRIPTION
**Pull request by ifoukarakis**
# Summary

Revised explores for DAU/MAU/WAU. The new explore allows:
- More performant query of the metrics
- Breakdown by server version and various server metadata (database, OS etc)

# Definitions

- DAU: Total number of unique user ids for the given day
- WAU: Total number of unique user ids for the given day and past 6 days (7 day rolling window)
- MAU: Total number of unique user ids for the given day and past 29 days (30 day rolling window)

# Implementation details

- The explore was added in a separate file. This is considered as a best practice.
- Drill fields have not yet specified. We can add them gradually as we discuss use cases.

### CODE REVIEW

File: views/marts/product/dim_daily_server_info.view.lkml [hunk 1]
```diff
@@ -8,14 +8,14 @@ view: dim_daily_server_info {
 
   dimension: daily_server_id {
     type: string
-    sql: ${TABLE}."daily_derver_id" ;;
+    sql: ${TABLE}.daily_derver_id ;;
```
Comment (catalintomai) [inline views/marts/product/dim_daily_server_info.view.lkml:11]: daily_**d**erver_id ?

Comment (ifoukarakis) [reply views/marts/product/dim_daily_server_info.view.lkml]: Typo, fixed.

**Review by sri-byte — APPROVED** [review]

File: views/marts/product/dim_daily_server_info.view.lkml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    sql: ${TABLE}.has_diagnostics_data ;;
+    label: "Has diagnostics data?"
+    description: " Whether the server queried the security update endpoint at the current date"
+    view_label: "- Data Source Info"
+  }
+
+  dimension: has_legacy_telemetry_data {
+    type: yesno
+    sql: ${TABLE}.has_legacy_telemetry_data ;;
+    label: "Has legacy telemetry"
```
Comment (catalintomai) [inline views/marts/product/dim_daily_server_info.view.lkml:156]: Does the "?" need to be part of the label for a `yesno` type or is on a case by case?

File: views/marts/product/dim_daily_server_info.view.lkml [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    description: "Whether telemetry data were reported from legacy telemetry pipeline (Segment) at the current date"
+    view_label: "- Data Source Info"
+  }
+
+  dimension: has_telemetry_data {
+    type: yesno
+    sql: ${TABLE}.has_telemetry_data ;;
+    label: "Has telemetry?"
+    description: "Whether telemetry data were reported from telemetry pipeline (Rudderstack) at the current date/"
```
Comment (catalintomai) [inline views/marts/product/dim_daily_server_info.view.lkml:165]: extra "\" suffix?

File: views/marts/product/dim_daily_server_info.view.lkml (hunk 2 above)
Comment (ifoukarakis) [reply views/marts/product/dim_daily_server_info.view.lkml:156]: Most likely a typo. We can decide if we like it or not and adjust. I'll add it for now just to be consistent, and we can revise later.

File: views/marts/product/dim_daily_server_info.view.lkml (hunk 3 above)
Comment (ifoukarakis) [reply views/marts/product/dim_daily_server_info.view.lkml:165]: Typo. Was meant to be a `.`. Fixing.

**Pull request merged**